spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -455,23 +455,23 @@ of the spec as laid out in this document.
 === LRA Client API
 
 For completeness the proposal supports clients that wish to directly
-control LRAs and participants. To support this class of user an instance of
+control LRAs and participants. To support this, the class of user can obtain an instance of
 <<source-LRAClient,LRA Client API>>
-can be instantiated directly or injected if the client is using CDI. This
+which can be instantiated directly or injected if the client is using CDI. This
 interface gives developers complete programmatic control of LRAs.
 
 [[java-based-lra-participant-registration-api]]
 === Java based LRA participant registration API
 
 If an application does not directly expose JAX-RS endpoints for
 compensation activities then participants can join an LRA directly using an instance of
-<<source-LRAManagement,LRAManagement>>
-This interface requires that a participant be a serializable java class conforming to the
-<<source-LRAParticipant,LRAParticipant interface>>
+<<source-LRAManagement,LRAManagement>>.
+This interface requires that a participant is a serializable java class conforming to the
```

Comment:
Not sure why you changed the original text but okay.

Reply:
@mmusgrov sorry, I am not a native speaker, but `...requires that a participant be...` doesn't sound right to me, would you prefer to remove this change?

Reply:
Personally I think either would be fine. I think the periods after the <<source.. make sense to add